live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js

```diff
@@ -0,0 +1,12 @@
+const deListFormatter1 = new Intl.ListFormat('de-DE', { style: 'short', type: 'disjunction' });
+const options1 = deListFormatter1.resolvedOptions();
+
+console.log(options1.locale);
+// expected output (Firefox / Safari): "de-DE"
+// expected output (Chrome): "de"
```

Comment:
I don't really like calling out specific browsers in here. Maybe we could avoid this issue by choosing `"fr"` instead? If so we should change the name of the variable of course.

Reply:
Oh I just saw that the other one uses `de`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/resolvedOptions. Well maybe keep it then, so we are consistent.

Reply:
I'll leave the language as it is then 👍